delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/eventlog/GlobalEventLog.scala

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.eventlog
+
+import akka.persistence.query.Offset
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Label
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.OrganizationRejection.OrganizationNotFound
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRejection.ProjectNotFound
+import fs2.Stream
+import monix.bio.{IO, Task}
+
+trait GlobalEventLog[T] {
+
+  /**
+    * Get stream of all [[Event]]s
+    * @param offset the offset to start from
+    */
+  def events(offset: Offset): Stream[Task, T]
```

Comment:
I changed to to `apply` because it seems enough for me. You're not fetching events anymore, you're fetching `T`

Reply:
I would rename it to `stream`, seems to give more information than just `apply`.